migrations/2023-08-24-203308_community_follower_notifications_enabled/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE community_follower
```

Comment:
Since notifications are a local-only / local user settings matter, this needs to be based on the `local_user` table, not `community_follower`, which is based on the `person` table (which may hold external users)

So you need to add a new table called `local_user_community_notification`